src/wymeditor/parser/xhtml-validator.js

```diff
@@ -21,13 +21,21 @@ WYMeditor.XhtmlValidator = {
             "attributes":[
             "class",
             "id",
-            "style",
```

Comment:
I've removed the `style` attribute from being valid on all elements except those listed under `except`, above.